cluster/examples/kubernetes/ceph/upgrade-from-v1.2-crds.yaml

```diff
@@ -0,0 +1,479 @@
+# Latest custom resource definitions
```

Comment:
Did you delete the upgrade-from-v1.1-crds.yaml as well? We shouldn't need that anymore.